plugins/css.js

```diff
@@ -1,6 +1,6 @@
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
-module.exports = ({devServer}) => !devServer && new MiniCssExtractPlugin({
+module.exports = ({devServer}) => !devServer ? new MiniCssExtractPlugin({
   filename: '[name].[hash].css',
   chunkFilename: '[id].[hash].css',
-});
+}) : undefined;
```

Comment:
I was checking for falsy values previously. I agree that we need 1 format (either a plugin or something else), but maybe it makes a bit more sense to return `null` instead of `undefined`.